lampod/src/actions/handler.rs

```diff
@@ -131,14 +135,23 @@ impl Handler for LampoHandler {
                 push_msat,
                 channel_type,
             } => {
-                Err(error::anyhow!("Request for open a channel received, unfortunatly we do not support this feature yet."))
+                log::info!("Recieved open channel request!");
+                let liquidity_manager = self.liquidity_manager.clone();
+                if let Some(_) = liquidity_manager {
+                    let _ = self.channel_manager.channeld().accept_inbound_channel_from_trusted_peer_0conf(&temporary_channel_id, &counterparty_node_id, 0);
+                }
```

Comment:
I do not think this is called at any point. We should keep the error here

Reply:
It is being called when we specifically accepting the channels with `0conf`, which would be used when we are a client requesting for in bound liquidity. This would come handy when the payer pays the JIT invoice and the LSP on the go creates a channel with the client.